doc/sedcli.8

```diff
@@ -81,6 +81,15 @@ Changes the lock state for the device. Following access modes are available:
 .IP "\fB\-P -d <device>\fR or \fB\-\-set-password --device <device>\fR"
 Updates password for Admin1 authority in Locking SP.
 
+.IP "\fB\-M -d <device> -e <enable> -m <done>\fR or \fB\-\-mbr-control --device <device> --enable {TRUE|FALSE} --done {TRUE|FALSE}\fR"
+.IP
+Enable/Disable MBR Shadow and Set/Unset MBR Done.
+MBR Done is TRUE only when the MBR Shadow is Enabled.
+
+.IP "\fB\-W -d <device> -f <File path containing Pre-boot Application(PBA) image> [-o]\fR or \fB\-\-write-mbr --device <device> --file <File path containing Pre-boot Application(PBA) image> [--offset]\fR"
```

Comment:
just "<file>" or <pba_file> would work better here

Reply:
I agree on that